sdk/python/kfp/containers/_component_builder.py

```diff
@@ -351,7 +355,11 @@ def build_python_component(
     local_requirement_filepath = os.path.join(local_build_dir, arc_requirement_filename)
     if is_v2:
       # For v2 components, KFP are expected to be packed in the container.
-      dependency.append(VersionedDependency(name='kfp', min_version='1.4.0'))
+      dependency.extend([
+          VersionedDependency(name='kfp', min_version='1.4.1rc0'),
+          VersionedDependency(name='absl-py', min_version='0.9', max_version='0.11'),
```

Comment:
Won't these be installed in any case?
Perhaps you can add a comment that describes why we're pinning these dependencies, but not the othe rones.

Reply:
Good point. I did hit a module not found error on `absl-py` when I tested this. So I just added all third party libs used in `entrypoint.py`. 
But now thinking back, I understand why it happened, kfp was missing `absl-py` at some point. It was fixed by https://github.com/kubeflow/pipelines/pull/5097/ shortly after 1.4.0 release. The first time I tested this, 1.4.1rc0 wasn't available yet.